src/kOS/Binding/TerminalSettings.cs

```diff
@@ -10,6 +11,7 @@ public override void AddTo(SharedObjects shared)
             shared.BindingMgr.AddGetter("SESSIONTIME", () => shared.Cpu.SessionTime);
             shared.BindingMgr.AddGetter("VERSION", () => Core.VersionInfo);
             shared.BindingMgr.AddGetter("VOLUME:NAME", () => shared.VolumeMgr.CurrentVolume.Name);
+            shared.BindingMgr.AddGetter("TERMINAL", () => new TerminalStruct(shared));
```

Comment:
Who knows :P i am willing to bet i added at least one of them but you are right. they belong somewhere else. so much so that i moved version to universe.